src/tiles/grid/item-tile.ts

```diff
@@ -223,19 +218,21 @@ export class ItemTile extends LitElement {
 
       .item-creator {
         color: #2c2c2c;
-        display: -webkit-box;
         font-size: 1.4rem;
         height: 3rem;
-        margin: 0px;
         overflow: hidden;
-        padding: 0.5rem;
         text-align: center;
         text-overflow: ellipsis;
         -webkit-line-clamp: 2;
         -webkit-box-orient: vertical;
+        display: flex;
+        justify-content: center;
+        align-items: flex-end;
+        padding-top: 1rem;
       }
 
-      #text-by {
+      .item-creator span:before {
+        content: 'By: ';
```

Comment:
This feels wrong to put semantically meaningful content in your CSS. What effect are you trying to achieve with this?

Reply:
i tried to separate the span text but got different positions on each so i did it this way.

is it better if i change it to: `<span><strong>By:</strong>${itemCreator}</span>`

Reply:
Yes, that would be better

Reply:
updated...